packages/portfolio-contract/src/pos-gmp.flows.ts

```diff
@@ -296,3 +296,51 @@ export const CompoundProtocol = {
     await sendGMPContractCall(target, calls, fee, lca, gmpChain);
   },
 } as const satisfies ProtocolDetail<'Compound', AxelarChain, EVMContext>;
+
+/**
+ * see {@link https://github.com/beefyfinance/beefy-contracts/blob/master/contracts/BIFI/vaults/BeefyVaultV7.sol }
+ * 0878a68 Nov 2022
+ */
+type BeefyVaultI = {
+  deposit: ['uint256'];
+  withdraw: ['uint256'];
+};
+
+const BeefyVault: BeefyVaultI = {
+  deposit: ['uint256'],
+  withdraw: ['uint256'],
+};
+
+export const BeefyProtocol = vaultName =>
```

Comment:
```suggestion
export const makeBeefyProtocol = vaultName =>
```

but I'm uneasy about making a new protocol impl each time.

I'd rather see vaultName moved to the `CTX`; see below...